src/iotconnect_discovery.c

```diff
@@ -76,6 +76,7 @@ IotclDiscoveryResponse *iotcl_discovery_parse_discovery_response(const char *res
         goto cleanup;
     }
 
+    if(base_url_cjson != NULL)
```

Comment:
Minor: The if opening curly brace should be in the same line as if. Block comment should be removed.